frontend/app/controllers/Event.scala

```diff
@@ -49,7 +49,7 @@ class Event(
 
     override def invokeBlock[A](request: Request[A], block: (Request[A]) => Future[Result]): Future[Result] = {
       eventbriteService.getEvent(eventId).map { event =>
-        SafeLogger.info(s"Buy action invoked for event: ${event.id}")
+        SafeLogger.info(s"Buy action invoked for event: ${event.underlying.ebEvent.id}")
```

Comment:
there was originally an implicit converion from RichEvent to Event, which I've removed.
https://github.com/guardian/membership-frontend/pull/1929/files#diff-3241cd55902f5775708482880f94420bL186
We have the extra layer because we have to group together the Description and Event objects.  This extra layer is because we expose the underlying wire-data model which has changed.
I thought about copying the new description into the old event description, but that seemed bodgetastic and I'd like to keep the general architecture that already exists where possible.
Another option would be to flesh out a couple more implicit conversions, but they are generally to be avoided e.g. https://dotty.epfl.ch/docs/reference/contextual/motivation.html